modules/installation-azure-service-principal.adoc

```diff
@@ -177,14 +175,16 @@ output. You need these values during {product-title} installation.
 +
 [source,terminal]
 ----
-$ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
+$ az ad sp create-for-rbac --role Contributor --name <service_principal> \ <1>
+  --scopes /subscriptions/<id>
```

Comment:
@MayXuQQ 
Can you check the 2nd line of this command? I assume that the scope is subscriptions. Does it need a resource group as well?